### ISSUE TITLE
Abrupt change to form and scope of panels

### ISSUE THREAD

**Issue by justinwb** [solid/process#127]
The scope and form of active panels [was changed](https://github.com/solid/process/pull/125) without consultation with panel stakeholders, despite the fact that there are active projects in flight (e.g. [data-interop](https://github.com/solid/data-interoperability-panel#projects)) that have now been seemingly split between new panels. There was no pull request for conversation about this change. There was no period of wait. There was no constructive deliberation about how things should be combined that included all of the necessary stakeholders. There is nowhere in our process that says that the scope and form of panels can be changed on a whim. People requested, volunteered, and participated in panels based on a given scope, it should be up to those same people to decide if they want to change that, not others.

Consequently, I have to officially note for the record my disagreement with [this change](https://github.com/solid/process/pull/125), and dissatisfaction with how it was made. 

I have to insist that this change be reverted, and resubmitted as a pull request so that it can be deliberated properly, out in the open, over an acceptable time period, with all of the relevant stakeholders.

**Comment by kjetilk** [comment]
Indeed, this was far too hasty, we need to have a PR we can discuss, in particular the splitting and redefinition of the data interop panel is problematic.

**Comment by RubenVerborgh** [comment]
+1 on reverting an discussing with panel stakeholders first

**Comment by justinwb** [comment]
@Mitzi-Laszlo and I chatted directly and agreed to a revert of #125 for now, noting that we both acknowledge further discussion regarding the reasons driving the changes is necessary. Based on that, and the support in this thread to roll things back - I have [committed the revert](https://github.com/solid/process/commit/376f1b09bcc4424839198b341ec7bb30aee7c671).

I'm going to close this issue now, since the specific reason it was created - changes to panels - has been resolved. That said, there is worthwhile and healthy dialogue that can and should happen to realign around some of the reasons for the change, and reasons for resistance to the same. That may happen in new pull requests, or in a separate issue altogether.
